File: extruturadedados/REVISÃO 3/REVISÃO/Repetição/COM_Repeticao_adivinha_numero.py
import random


segredo = random.randint(1, 100)
tentativas = 0

print("🎲 Adivinhe o número entre 1 e 100!")

while True:
    chute = int(input("Digite seu palpite: "))
    tentativas += 1

    if chute > segredo:
        print("Muito alto!\n")
    elif chute < segredo:
        print("Muito baixo!\n")
    else:
        print(f"🎉 Acertou em {tentativas} tentativas!")
        break

File: extruturadedados/REVISÃO 3/REVISÃO/Repetição/COM_Repeticao_adivinha_numero.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    srand(time(0));
    int segredo = rand() % 100 + 1;  // número entre 1 e 100
    int chute, tentativas = 0;

    printf("🎲 Adivinhe o número entre 1 e 100!\n");

    do {
        printf("Digite seu palpite: ");
        scanf("%d", &chute);
        tentativas++;

        if (chute > segredo)
            printf("Muito alto!\n");
        else if (chute < segredo)
            printf("Muito baixo!\n");
        else
            printf("🎉 Acertou em %d tentativas!\n", tentativas);
    } while (chute != segredo);

    return 0;
}
